static/js/components/forms/EditProfileForm_test.js

```diff
@@ -61,7 +61,7 @@ describe("EditProfileForm", () => {
     wrapper.update()
     assert.deepEqual(
       findFormikErrorByName(wrapper, "legal_address.street_address").text(),
-      "Street address is a required field"
+      "Street address field must have at least 1 items"
```

Comment:
This is a less helpful error message than what we had before. Can you look into changing it?

Reply:
sure.

Reply:
Sorry, I re requested you the change, let me update it then tagged you again.